src/documents/tutorial-prefsFrameworkMinimalFootprint/MinimalFootprint.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Preferences Framework: Minimal Footprint
+layout: default
+category: Tutorials
+---
+
+Website performance can be split into several buckets: backend, over the wire, and front end. It is important to pay attention to all of these areas when devising your optimization strategy. This document will focus on using the [Preferences Framework](../PreferencesFramework.md) in a situation where optimization is critically important; however, it will not go into detail to cover all of the potential optimizations that can be employed on your site.
```

Comment:
This paragraph is self-contradictory. In the first sentence you explain that there are several "buckets" - in the second sentence you simply talk of "optimization" without distinguishing between buckets. "critically important" is pretty vague